charts/projectsveltos/templates/deployment.yaml

```diff
@@ -32,17 +32,34 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       containers:
-{{- if .Values.agent.managementCluster }}      
-      - args: {{- toYaml .Values.addonController.controller.argsAgentMgmtCluster | nindent 8 -}}
-{{ else }}
-      - args: {{- toYaml .Values.addonController.controller.args | nindent 8 -}}
-{{- end }}
-{{- if ne (len .Values.addonController.driftDetectionManagerPatchConfigMap.data) 0 }}
-        - --drift-detection-config={{ .Values.addonController.driftDetectionManagerPatchConfigMap.name }}
-{{- end }}
-{{- if .Values.telemetry.disabled }}
+  {{- if .Values.agent.managementCluster }}
+      - args:
+      {{- range .Values.addonController.controller.argsAgentMgmtCluster }}
+        - {{ . }}
+      {{- end }}
+      {{- range $key, $value := .Values.addonController.controller.extraArgsAgentMgmtCluster }}
```

Comment:
shouldn't we add the Values.addonController.controller.extraArgsAgentMgmtCluster part? it appears we are removing the Values.addonController.driftDetectionManagerPatchConfigMap.data